src/cmd/manage_nsfs.js

```diff
@@ -143,7 +143,8 @@ async function fetch_bucket_data(action, user_input) {
         path: user_input.path,
         should_create_underlying_storage: action === ACTIONS.ADD ? false : undefined,
         new_name: _.isUndefined(user_input.new_name) ? undefined : String(user_input.new_name),
-        fs_backend: _.isUndefined(user_input.fs_backend) ? config.NSFS_NC_STORAGE_BACKEND : String(user_input.fs_backend)
+        fs_backend: _.isUndefined(user_input.fs_backend) ? config.NSFS_NC_STORAGE_BACKEND : String(user_input.fs_backend),
+        force_md5_etag: _.isUndefined(user_input.force_md5_etag) ? undefined : get_boolean_or_string_value(user_input.force_md5_etag)
```

Comment:
Maybe you can eliminate the unset ('') here instead of lines 173-174, for example:
```suggestion
        force_md5_etag: _.isUndefined(user_input.force_md5_etag) || user_input.force_md5_etag === ''? undefined : get_boolean_or_string_value(user_input.force_md5_etag)
```
* same comment in `fetch_account_data`.

Reply:
I don't think it will work here since if we put an undefined value here it will be overridden by fetch_existing_bucket_data. also, I feel like lines 173-174 is a better place since it is where we unset fs_backend and s3_policy

Reply:
I can understand that you prefer to do the override in the same section where we unset fs_backend and s3_policy.
I find it confusing that in the unset of `force_md5_etag` we are using the `user_input.force_md5_etag` in condition and not `data.force_md5_etag`.